cratedb_toolkit/util/database.py

```diff
@@ -325,3 +324,21 @@ def decode_database_table(url: str) -> t.Tuple[str, str]:
         if url_.scheme == "crate" and not database:
             database = url_.query_params.get("schema")
     return database, table
+
+
+def quote_table_name(name: str) -> str:
+    """
+    Quote table name if not happened already.
+
+    In:  foo
+    Out: "foo"
+
+    In:  "foo"
+    Out: "foo"
+
+    In:  foo.bar
+    Out: foo.bar
+    """
+    if '"' not in name and "." not in name:
+        name = f'"{name}"'
+    return name
```

Comment:
@seut: Do you think this routine needs to be improved? You mentioned something about "quoting going south". Maybe the root cause is here.

Reply:
Thank you. I've diverted that discussion to a separate backlog item. It will be an easy but valuable improvement.

- GH-168